packages/dapi/test/unit/grpcServer/handlers/blockheaders-stream/subscribeToBlockHeadersWithChainLocksHandlerFactory.spec.js

```diff
@@ -148,27 +149,29 @@ describe('subscribeToBlockHeadersWithChainLocksHandlerFactory', () => {
 
     call = new GrpcCallMock(this.sinon, request);
 
-    call.request.setFromBlockHash('someBlockHash');
+    const blockHash = '00000bafbc94add76cb75e2ec92894837288a481e5c005f6563d91623bf8bc2c';
+
+    call.request.setFromBlockHash(Buffer.from(blockHash, 'hex'));
```

Comment:
It must be Uint8Array not a Buffer

Reply:
I'm not sure that much important in this case, because converting string to uint8 is not trivial in nodejs and it's all about .toString('hex') in the implementation

Reply:
Yeah, but if we won't do `Buffer.from` it won't work, right?

Reply:
Yes, because we expect Uint8Array from the grpc request. Buffer is a subclass of the Uint8array though

Reply:
It was working before because there weren't any assertions on block hash. I've added it in this pr